style.css

```diff
@@ -89,4 +89,209 @@
+    grid-template-columns: 327px;
+    grid-template-rows: 474px 474px 474px;
+    grid-gap: 24px;
+    align-items: center;
+  }
+
+  .entries {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .entries img {
+    border-radius: 8px 8px 0 0;
+  }
+
+  .description {
+    background: #fff;
+    align-items: center;
+    border-radius: 0 0 8px 8px;
+  }
+
+  .description h4 {
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: center;
+    padding: 0;
+    margin: 10px;
+  }
+
+  .description ul {
+    margin: 0, 11px;
+    display: flex;
+    flex-grow: 1;
+    justify-items: space-between;
+  }
+
+  .list1 li {
+    background: #ebf0ee;
+    border-radius: 4px;
+    margin: 0 5px 0 5px;
+    padding: 5px;
+    font-size: 12px;
+    font-weight: 600;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    text-align: left;
+  }
+}
+
+.button-1 {
+  color: white;
+  height: auto;
+  width: 98px;
+  left: 12px;
+  top: 12px;
+  margin-left: 103px;
+  margin-bottom: 10px;
+  border-radius: 4px;
+  background: #36b37f;
+}
+
+.about-me {
+  background-image: url(about-me-1.svg), url(about-me-2.svg);
+  background-position: right top, left bottom;
+  background-repeat: no-repeat, no-repeat;
+  display: flex;
+  flex-direction: column;
+}
+
+.about-me h2 {
+  margin: 0;
+}
+
+#bio {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.button-2 {
+  color: white;
+  width: 135px;
+  height: 24px;
+  margin-left: 103px;
+  margin-bottom: 10px;
+  background: #36b37f;
+}
+
+.skills {
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-rows: 352px 352px 352px;
+  grid-gap: 24px;
+  align-items: center;
+}
+
+.skill-entries {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+}
+
+.icons {
+  margin-top: 24px;
+  height: 80px;
+  width: 80px;
+  left: 124px;
+  top: 24px;
+  border-radius: 0;
+}
+
+.title {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.skill-entries ul {
+  margin: 0, 11px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-items: center;
+  align-content: center;
+}
+
+.list2 li {
+  color: #36b37e;
+  background: #fff;
+  border-radius: 8px;
+  margin: 5px;
+  padding: 10px;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
+
+.form {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto;
+  justify-items: center;
+  grid-gap: 24px;
+}
+
+.form-entries {
+  display: grid;
+  grid-template-columns: 340px;
+  grid-template-rows: auto auto;
+  grid-gap: 24px;
+}
+
+#submit {
+  padding: 12px;
+  text-align: justify;
+}
+
+#emai {
+  font-size: 20px;
+  font-weight: 400;
```

Comment:
- [ ] Kindly make sure you stick to the design. the typo, color, socing(margin, padding) are supposed to be similar to the original design
`Hint: To get a element style, just click it and you will see the style on the right side in Figma`
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205438231-d40c8f8b-9b73-4530-a013-80323305f4d2.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205438251-ded0697a-56f9-4759-bba1-a49ed8d76456.png)